packages/ckb/src/spore/spore.ts

```diff
@@ -205,13 +202,47 @@ export const appendIssuerCellToSporesCreate = async ({
   rawTx.outputs = [...rawTx.outputs, changeOutput];
   rawTx.outputsData = [...rawTx.outputsData, '0x'];
 
-  rawTx.cellDeps = [...rawTx.cellDeps, getSecp256k1CellDep(isMainnet)];
+  if (witnessLockPlaceholderSize === SECP256K1_WITNESS_LOCK_SIZE) {
+    rawTx.cellDeps = [...rawTx.cellDeps, getSecp256k1CellDep(isMainnet)];
```

Comment:
Perhaps this line of code could be moved to `fn appendIssuerCellToSporesCreate`:L242

Reply:
> Perhaps this line of code could be moved to fn appendIssuerCellToSporesCreate:L242

This will lead to inaccurate fee estimates.

Judging which lock it belongs to based on the issuer lock seems to be a better choice

Reply:
Move the secp256k1 cell dep to signing stage in https://github.com/ckb-cell/rgbpp-sdk/pull/171/commits/ce21ad377477b13e3d25c4d52767ebf1c32ed2b1